hooks/post_gen_project.py

```diff
@@ -1,15 +1,95 @@
+import subprocess
+import shutil
 from pathlib import Path
 
 
-def check_testing_type():
+def clean_project():
     if "{{cookiecutter.testing_type}}" == "django":
         print("REMOVE: pytest.ini")
         Path(Path().cwd(), "pytest.ini").unlink()
 
+    if "{{cookiecutter.css_style}}" == "sass":
+        Path("./tailwind.config.js").unlink()
+        Path("./postcss.config.js").unlink()
+        Path("./{{cookiecutter.project_slug}}/assets/styles/tailwind_entry.css").unlink()
+        shutil.rmtree(Path("./apps/{{cookiecutter.project_slug}}/assets/styles/tailwind"))
+    else:
+        Path("./{{cookiecutter.project_slug}}/assets/styles/sass_entry.scss").unlink()
+        shutil.rmtree(Path("./apps/{{cookiecutter.project_slug}}/assets/styles/sass"))
```

Comment:
I had chosen a primary app name different from the default (and tailwind), and got an error on this `rmtree` due to the specified path not existing. Possibly this (and in the section above where tailwind stuff is removed?) should really be `{{cookiecutter.primary_app}}` (I'm not entirely sure that's the right identified.)